.github/workflows/ci.yml

```diff
@@ -0,0 +1,117 @@
+name: CI Checks
+
+on:
+  push:
+    branches: ["**"]
+  pull_request:
+    branches: [master]
+  workflow_dispatch:
+
+jobs:
+  demo-build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone This Repo
+        uses: actions/checkout@v2
```

Comment:
Instead of cloning the repository multiple times, is it possible to upload the cloned repository as a temporary artifact at the beginning and use it across all the jobs?

Reply:
The whole thing runs in under 5 minutes. Let's consider it if it hits 10 for now.